packages/local-cli/core/getCommands.js

```diff
@@ -0,0 +1,99 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
```

Comment:
so what is the final decision on the copyright headers?
We should solve it before 0.59

Reply:
I don't know, but will definitely remove the added ones.